tests/unittests/OperatorTest.cpp

```diff
@@ -69,6 +69,262 @@ static Tensor createTensorConditionallyQuantized(ElemKind T,
 }
 
+template <typename DataType>
+glow::Handle<bool>
+lessHelper(glow::PlaceholderBindings &bindings, glow::Module &mod,
+           glow::Function *F, glow::ExecutionEngine &EE, ElemKind DTy,
+           llvm::ArrayRef<DataType> xValues, llvm::ArrayRef<DataType> yValues,
+           llvm::ArrayRef<size_t> xDims, llvm::ArrayRef<size_t> yDims) {
+  auto *X = createPlaceholderConditionallyQuantized(mod, DTy, xDims, "X",
+                                                    DTy != ElemKind::FloatTy);
+
+  auto *Y = createPlaceholderConditionallyQuantized(mod, DTy, yDims, "Y",
+                                                    DTy != ElemKind::FloatTy);
+
+  bindings.allocate(llvm::dyn_cast<Placeholder>(X))->getHandle<DataType>() =
+      xValues;
+
+  bindings.allocate(llvm::dyn_cast<Placeholder>(Y))->getHandle<DataType>() =
+      yValues;
+
+  auto *cmpr = F->createNodeWithBroadcast<CmpLTNode>("cmpLT", -1, X, Y);
+
+  auto *save = F->createSave("save", cmpr);
+  auto *saveAlloc = bindings.allocate(save->getPlaceholder());
+
+  EE.compile(CompilationMode::Infer);
+  EE.run(bindings);
+
+  return saveAlloc->getHandle<bool>();
+}
+
+TEST_P(OperatorTest, less_int8) {
+  ENABLED_BACKENDS(Interpreter, CPU);
+
+  llvm::SmallVector<int8_t, 16> xValues = {
+      3, 2, 3, 6, 4, 5, 6, 3, 7, 8, 9, 2, 3, 5, 7, 1,
+
+      1, 2, 3, 6, 4, 5, 6, 3, 7, 8, 9, 2, 3, 5, 7, 1,
+
+      1, 2, 3, 6, 4, 5, 6, 3, 7, 8, 9, 2, 3, 5, 7, 1,
+
+      1, 2, 3, 6, 4, 5, 6, 3, 7, 8, 9, 2, 3, 5, 7, 1};
+
+  llvm::SmallVector<int8_t, 16> yValues = {
+      3, 4, 5, 7, 2, 5, 0, 6, 4, 2, 1, 8, 5, 9, 2, 6,
+
+      3, 4, 5, 7, 2, 1, 0, 6, 4, 2, 1, 8, 5, 9, 2, 6,
+
+      3, 4, 5, 7, 2, 1, 0, 6, 4, 2, 1, 8, 5, 9, 2, 6,
+
+      3, 4, 5, 7, 2, 1, 0, 6, 4, 2, 1, 8, 5, 9, 2, 6};
+
+  llvm::SmallVector<size_t, 4> xDims = {2, 2, 4, 4};
+  llvm::SmallVector<size_t, 4> yDims = {2, 2, 4, 4};
+
+  Handle<bool> saveH =
+      lessHelper<int8_t>(bindings_, mod_, F_, EE_, ElemKind::Int8QTy, xValues,
+                         yValues, xDims, yDims);
+
+  llvm::SmallVector<bool, 16> refResults = {
+      false, true,  true,  true, false, false, false, true,
+      false, false, false, true, true,  true,  false, true,
+
+      true,  true,  true,  true, false, false, false, true,
+      false, false, false, true, true,  true,  false, true,
+
+      true,  true,  true,  true, false, false, false, true,
+      false, false, false, true, true,  true,  false, true,
+
+      true,  true,  true,  true, false, false, false, true,
+      false, false, false, true, true,  true,  false, true,
+  };
+
+  int counter = 0;
+  for (size_t i = 0; i < saveH.dims()[0]; ++i) {
+    for (size_t j = 0; j < saveH.dims()[1]; ++j) {
+      for (size_t k = 0; k < saveH.dims()[2]; ++k) {
+        for (size_t f = 0; f < saveH.dims()[3]; ++f) {
+          EXPECT_FLOAT_EQ(refResults[counter++], saveH.at({i, j, k, f}));
+        }
+      }
+    }
+  }
+}
+
+TEST_P(OperatorTest, less_floatCases) {
```

Comment:
What signal does this test provide that `less_float` doesn't?

Reply:
Tests all three cases for cmp explicitly on a small dataset, without broadcast. I can remove.